samples/net/rpl_border_router/sample.yaml

```diff
@@ -0,0 +1,8 @@
+sample:
+    name: RPL Border Router
+tests:
+-   test:
+        build_only: true
+        platform_whitelist: frdm_k64f qemu_x86
```

Comment:
I left the qemu_x86 there so it gets compile tested in qemu. Probably this is not needed if we are not running it in qemu.
I will add the depends_on line as suggested.